debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/SnapshotReader.java

```diff
@@ -232,10 +249,18 @@ protected void execute() {
                         if (filters.tableFilter().test(id)) {
                             tableIds.add(id);
                             tableIdsByDbName.computeIfAbsent(dbName, k -> new ArrayList<>()).add(id);
+                        } else if (logger.isTraceEnabled()) {
+                            logger.trace("'{}' is filtered out, discarding", id);
```

Comment:
How about change this to debug-level, and indent a bit so its more obvious they are within Step 5?